proposals/xxxx-work-graphs.md

```diff
@@ -0,0 +1,551 @@
+They also have new annotations unique to node shaders.
+
+> Consistent with HLSL grammar, attribute names are case-insensitive. Any
+> attributes that take string arguments, the string argument values are
+> case-sensitive.
+
+##### **`[NodeLaunch("<mode>")]`**
+Valid values for node launch mode are `broadcasting`, `coalescing`, or `thread`.
+If the `NodeLaunch` attribute is not specified on a node entry the default
+launch mode is `broadcasting`.
+
+##### **`[NodeIsProgramEntry]`**
+
+
+Indicates a node that may be invoked as an entry directly by the API.
+Shaders that can receive input records from both inside and outside the work
+graph (i.e. from a command list) require this attribute. Graphs can have more
+than one entry shader that receives inputs from outside the graph.
+This property is implied if no other work graph nodes target this node,
+so in that case the attribute is optional and may be omitted.
+
+##### **`[NodeID("<name>", <index> = 0)]`**
+If present the shader represents a node with the provided name. If present, the
+index parameter specifies the index into the named node array. If this attribute
+is not present, the function name is the node name, and the index is `0`.
+
+##### **`[NodeLocalRootArgumentsTableIndex(<index>)]`**
+The specified index indicates the record index into the local root arguments
+table bound to the work graph. If this attribute is not specified and the shader
+has a local root signature, the index defaults to an unallocated table location.
+
+##### **`[NodeShareInputOf("<name>", <index> = 0 )]`**
+Share the inputs for the node specified by name and optional index with this
+node. Both nodes must have identical input records, the same launch mode, and
+identical dispatch grid size (if fixed).
+
+##### **`[NodeDispatchGrid(<x>, <y>, <z>)]`**
+Specifies the size of the dispatch grid. Broadcast launch nodes must specify
+either the dispatch grid size or maximum dispatch grid size in source. The `x`
+`y` and `z` parameters individually cannot exceed (2^16)-1 (65535), and `x*y*z`
+cannot exceed (2^24)-1 (16,777,215).
+
+##### **`[NodeMaxDispatchGrid(<x>, <y>, <z>)]`**
+Specifies the maximum dispatch grid size when the dispatch grid size is
+specified on the input record using the `SV_DispatchGrid` semantic. The `x` `y`
+and `z` parameters individually cannot exceed (2^16)-1 (65535), and `x*y*z` cannot
+exceed (2^24)-1 (16,777,215).
+
+##### **`[NodeMaxRecursionDepth(<count>)]`**
+Specifies the maximum recursion depth for a node. This attribute is required if
+one of the outputs of a node is the ID of the node itself.
+
+#### Entry Parameters
+
+Specific types for input parameters depend on the node launch mode, but all node
+shaders support two categories of parameters:
+
+* Zero or one [Node Input Parameter](#node-input-objects).
+* Zero or more [Node Output Parameter(s)](#node-output-objects).
+
+[Broadcast and Coalescing Launch](launch-modes) shaders also support optional
+system value parameters.
```

Comment:
I think you misunderstood my previous comment, which I'll resolve since it's pointing to the old text anyway.  I meant that this spec is missing a section describing the allowed system value parameters.

Reply:
I've lightly modified the corresponding section from the API spec and moved it to its own section after the inputs documentation.